packages/wonder-blocks-dropdown/src/components/__tests__/action-item.test.js

```diff
@@ -1,33 +1,53 @@
+// @flow
 import * as React from "react";
-import renderer from "react-test-renderer";
 import {MemoryRouter, Link} from "react-router-dom";
 import {mount} from "enzyme";
-import "jest-enzyme";
+import {render, screen} from "@testing-library/react";
 
 import ActionItem from "../action-item.js";
 
 describe("ActionItem", () => {
     it("should render with disabled styles", () => {
-        const actionItem = renderer.create(
-            <ActionItem href="/foo" disabled={true} />,
-        );
+        // Arrange
+
+        // Act
+        render(<ActionItem href="/foo" label="Example" disabled={true} />);
 
-        expect(actionItem).toMatchSnapshot();
+        // Assert
+        expect(screen.getByRole("menuitem")).toBeDisabled();
     });
 
     it("should render an anchor if there's no router", () => {
-        const wrapper = mount(<ActionItem href="/foo" />);
+        // Arrange
 
+        // Act
+        const wrapper = mount(<ActionItem href="/foo" label="Example" />);
+
+        // Assert
         expect(wrapper.find("a")).toHaveLength(1);
     });
 
     it("should render a Link if there's a router", () => {
+        // Arrange
+
+        // Act
         const wrapper = mount(
             <MemoryRouter>
-                <ActionItem href="/foo" />
+                <ActionItem href="/foo" label="Example" />
             </MemoryRouter>,
         );
 
+        // Assert
```

Comment:
Same thing here because RTL cannot query components. I think that's what's ADR 526 is about.